cfme/tests/configure/test_start_page.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+import fauxfactory
+
+from cfme.base.credential import Credential
+from cfme.configure.settings import Visual
+from cfme.configure.access_control import Group, User
+from cfme.utils.appliance.implementations.ui import navigate_to
+
+
+faux_value = fauxfactory.gen_alphanumeric()
+pytestmark = pytest.mark.usefixtures('browser')
+
+
+def get_landing_page():
```

Comment:
**REQUIRED:** Though it's clever, I'm not a fan of dynamic parametrization. The only "dynamic" parameter we have right now is ```provider``` and as we have discussed in the team calls, we are going to get rid of it in favour of statically defined parameters.

There are reasons for this
1) Parameters link to test cases, and so making them dynamic means you are not sure what your test plan really is
2) They can be screwed up and tricked, leading to bad test cases being generated, particularly for the last case where these are parametrized.
3) This requires that the landing pages are read at collection time. This is a **huge** no no. It means that we MUST have an appliance in order to do collection which is simply not allowed. This should be switched to a static list of landing pages.